src/Core/src/Platform/Windows/MauiWinUIWindow.cs

```diff
@@ -178,6 +199,11 @@ internal void UpdateTitleOnCustomTitleBar()
 
 		[DllImport("user32.dll", SetLastError = true)]
 		static extern int DestroyIcon(IntPtr hIcon);
+
+		internal void SetWindow(IWindow window)
```

Comment:
The lifecycle hooks all reached the xplat window by iterating over the static applications windows. This doesn't work very well for the tests because the tests run their own stubbed app that windows get attached to. This removes the need to resort to static scope in order to get from the platform window to the xplat window